Automattic/jetpack
# Add admin notice to media-new.php regarding videopress uploads not being supported

Anecdotal: There used to be a warning at the top of `media-new.php` warning users that uploading from this page would bypass VideoPress and upload directly to their server. This message is now gone and has caused confusion for us and others as we test Jetpack + VideoPress.

This PR re-introduces this message, only when VideoPress is enabled.

<img width="751" alt="Screen Shot 2021-10-01 at 4 51 48 PM" src="https://user-images.githubusercontent.com/4081020/135684826-976ba6c3-77d8-46b7-84b2-a11454ea852e.png">





Fixes #

#### Changes proposed in this Pull Request:

* Added an admin notice to the top of `media-new.php` warning the user that VideoPress uploading is not supported here, only when VideoPress is enabled

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


Nope!

#### Testing instructions:






* On any Jetpack site, disable VideoPress (Jetpack -> Settings -> Performance)
* Visit Media -> Add New (not Library -> Add New)
* You should not see any admin notices.
* Enable VideoPress, then return to Media -> Add New
* You should now see the admin notice!
